gtsam/geometry/Rot3.cpp

```diff
@@ -32,7 +32,8 @@ namespace gtsam {
 
 /* ************************************************************************* */
 void Rot3::print(const std::string& s) const {
-  gtsam::print((Matrix)matrix(), s);
+  cout << (s.empty() ? "R: " : s + " ");
+  gtsam::print((Matrix)matrix());
```

Comment:
prefer `static_cast<>()` instead of old C cast?

Reply:
Yes! I was trying not to introduce too many changes, but this is a very good change to introduce.